torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2741,15 +2774,13 @@ def _make_tensor_helper(shape, low=None, high=None):
     OpInfo('index_select',
            dtypes=all_types_and_complex_and(torch.bool, torch.float16, torch.bfloat16),
            test_inplace_grad=False,
-           skips=(
-               # https://github.com/pytorch/pytorch/issues/49707
-               SkipInfo('TestCommon', 'test_variant_consistency_eager',
-                        dtypes=[torch.float16, torch.bfloat16]),
-               SkipInfo('TestCommon', 'test_variant_consistency_jit', dtypes=[torch.float16, torch.bfloat16]),
-               # index_select does not correctly warn when resizing out= inputs
-               SkipInfo('TestCommon', 'test_out')
-           ),
+           supports_out=False,  # index_select does not correctly warn when resizing out=
            sample_inputs_func=sample_inputs_index_select),
+    OpInfo('index_add',
+           dtypes=all_types_and_complex_and(torch.bool, torch.float16, torch.bfloat16),
+           test_inplace_grad=False,
```

Comment:
why is `test_inplace_grad` set to `False`?

Reply:
I think I just set it to false because the other `index_*` operations also did. But now that I look at the derivatives, this one does not make use of inplace operations, so I believe it should work with inplace grads. I will test this in a second.

Reply:
The tests seem to pass. Thank you for the catch!